### PR TITLE
Basic gameplay

### PR DESCRIPTION
**Pull request by robbertzzz**
Note pitches are highly likely to be off, but can't confirm until the input XML file contains something musical. Apart from that, the visual pitch and the input pitch do correspond with one another.

res://scenes/performance/notes.gd has a series of functions at the end where any further gameplay feedback can be implemented. For now they just print a message.
res://scenes/performance/note.gd has a few functions where visual effects for the note "blocks" can be implemented to provide further feedback.

### CODE REVIEW

File: scenes/song_parser/song.gd [hunk 1]
```diff
@@ -1,29 +1,18 @@
-extends Node
 class_name Song
 
 
-@export var artistName = ""
-@export var title = ""
-@export var artistNameSort = ""
-@export var artistYear = ""
-@export var artistXXXXYear = ""
```
Comment (mattkanwisher) [inline scenes/song_parser/song.gd:9]: I was getting compiler warnings before about, if stuff isnt exported you cant use this Song class effectively in other classes

Comment (robbertzzz) [reply scenes/song_parser/song.gd:9]: Do you know in what context that was? Export tooltips are only useful when you want to edit the values of a node or Resource in the inspector but we're doing neither of those things.

**Review by mattkanwisher — APPROVED** [review]

File: scenes/song_parser/note.gd [hunk 2]
```diff
@@ -1,39 +1,44 @@
-extends Node
-class_name SNote
+class_name Note
```
[the remark this answers is not in the document]
Comment (robbertzzz) [reply scenes/song_parser/note.gd:1]: I hated it too!

File: scenes/song_parser/song.gd (hunk 1 above)
Comment (robbertzzz) [reply scenes/song_parser/song.gd:9]: Additionally, @export doesn't work when a script isn't a Node or Resource, it's neither in this PR.

**Pull request merged**